examples/BuddyMatmul/linalg-transposematmulb-f32.mlir

```diff
@@ -0,0 +1,79 @@
+// RUN: buddy-opt %s \
+// RUN:     -matmul_transpose_b_vectorization \
```

Comment:
Recommend to change `-matmul_transpose_b_vectorization` to `-matmul-transpose-b-vectorization`